tests/fixtures/issues/issue-1323.php

```diff
@@ -57,15 +57,15 @@
     'issue-1323-B' => [
         'ua' => 'Mozilla/5.0 (iPhone; CPU iPhone OS 10_3 like Mac OS X) AppleWebKit/603.1.30 (KHTML, like Gecko) Mobile/14E5260b Version/10.3 Safari/8536.25 Mobicip/1947694464',
```

Comment:
After I added the mobicip browser, this one failed, it is mobicip though, so was an uncaught  mis-detection previously.